docs/en/latest/admin-api.md

```diff
@@ -278,6 +278,42 @@ curl 'http://127.0.0.1:9180/apisix/admin/routes?name=test&uri=foo&label=' \
 }
 ```
 
+### Support reference filtering query
+
+:::note
+
+This feature was introduced in APISIX 3.13.0.
```

Comment:
Don't think this sentence is necessary? This doc change will go into "next" and only become visible in 3.13.0 version once released.

I think we can get rid of this note and just put the sentence below in a regular paragraph.

> APISIX supports querying routes and stream routes by `service_id` and `upstream_id`. Other resources or fields are not currently supported.

WDYT?

Reply:
I did encounter this. Users do not take care to manually select the older version of the documentation they are using, and trying a particular new version of a supported feature on an older version directly from the documentation.

This is confusing.